lib/server_web/live/user_live/new_live.ex

```diff
@@ -58,15 +121,27 @@ defmodule ServerWeb.User.NewLive do
   end
 
   def handle_event("add_user", _params, %{assigns: %{user: user}} = socket) do
-    user = Validation.validate_on_submit(user, &Validators.validator_for/1)
+    user =
+      user
+      |> Validation.validate_on_submit("name", Validators.validator_for("name"))
+      |> Validation.validate_on_submit("surname", Validators.validator_for("surname"))
+      |> Validation.validate_on_submit("password", Validators.validator_for("password"))
+      |> Validation.validate_on_related_submit(
+        "conf_password",
+        "password",
+        Validators.validator_for("conf_password")
+      )
```

Comment:
Prosim te, related validace musi vzdy byt po validaci fieldu, na kterem zavisi. Jinak by to fungovalo blbe. Zatim to nedokazu zarucit nejak automaticky.

Reply:
Oki, beru v potaz